contrib/epee/include/net/net_utils_base.h

```diff
@@ -45,6 +42,7 @@
 #define MAKE_IP( a1, a2, a3, a4 )	(a1|(a2<<8)|(a3<<16)|(a4<<24))
 #endif
 
+namespace boost { namespace asio { class io_service; } }
```

Comment:
This needs to be reverted. Newer versions of boost have changed the name to `io_context` and then `typedef io_context io_service`. So there is no longer a `class io_service`. This _should_ happen in the next release (boost 1.66).